Tasks/Maven/sonarqube-common/sonarqube-rest.ts

```diff
@@ -0,0 +1,193 @@
+/// <reference path="../../../definitions/vsts-task-lib.d.ts" />
+
+import Q = require('q');
+import url = require('url');
+import http = require('http');
```

Comment:
It's too complex to use http / https on your own. Why not use a module such as "request" that hides this complexity away?